service/routes.py

```diff
@@ -286,7 +286,16 @@ def put(self, product_id, condition):
                 data["quantity"],
                 data["restock_level"],
             )
-            return "", status.HTTP_400_BAD_REQUEST
+            if data["quantity"] < 0:
+                abort(
+                    status.HTTP_400_BAD_REQUEST,
+                    f"Quantity is given as {data['quantity']}. It cannot be negative.",
+                )
+            else:
+                abort(
+                    status.HTTP_400_BAD_REQUEST,
+                    f"Restock is given as {data['restock_level']}. It cannot be negative.",
+                )
```

Comment:
Earlier there was no message indication on the UI incase of `DataValidationError` from backend. Added these `abort` statements to indicate the correct failure message on UI.